src/Network/Ricochet/Types.hs

```diff
@@ -53,8 +65,8 @@ data ChannelType = MkChannelType ByteString
 
 -- | A contact of our user, defined by his ID (TOR hidden service address without the .onion and the 'ricochet:' prefix) and his display name
```

Comment:
“Our user” sounds as if we would store information about the user that uses the client that is developed using this library.